DigitalLearningSolutions.Data.Migrations/202112091009_AddColumnsToCompetencyResourceAssessmentQuestionParameters.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace DigitalLearningSolutions.Data.Migrations
+{
+    using FluentMigrator;
+
+    [Migration(202112091009)]
+    public class AddColumnsToCompetencyResourceAssessmentQuestionParameters : Migration
+    {
+        public override void Up()
+        {
+            Alter.Table("CompetencyResourceAssessmentQuestionParameters")
+                .AddColumn("RelevanceAssessmentQuestionID").AsInt32().Nullable().WithDefaultValue(null)
+                .ForeignKey("AssessmentQuestions", "ID")
+                .AddColumn("CompareToRoleRequirements").AsBoolean().NotNullable().WithDefaultValue(false);
+        }
+
+        public override void Down()
+        {
+            Delete.ForeignKey(
+                    "FK_CompetencyResourceAssessmentQuestionParameters_RelevanceAssessmentQuestionID_AssessmentQuestions_ID"
+                )
+                .OnTable("CompetencyResourceAssessmentQuestionParameters");
+            Delete.Column("RelevanceAssessmentQuestionID").FromTable("CompetencyResourceAssessmentQuestionParameters");
+            Delete.Column("CompareToRoleRequirements ").FromTable("CompetencyResourceAssessmentQuestionParameters");

```

Comment:
Looks like an extra space after CompareToRoleRequirements

Reply:
Good spot. I tested the Down migration while it had the space, and the column was removed fine, so it looks like the Migrator is clever enough to trim these. I've removed the space now.